src/debug_menu.cpp

```diff
@@ -61,6 +76,123 @@ class mission_debug
         static std::string describe( const mission &m );
 };
 
+static int player_uilist()
+{
+    const std::vector<uilist_entry> uilist_initializer = {
+        uilist_entry( DEBUG_MUTATE, true, 'M', _( "Mutate" ) ),
```

Comment:
This can be written without repeating `uilist_entry` on every line:
```c++
const std::vector<uilist_entry> uilist_initializer = {
    { DEBUG_MUTATE, true, 'M', _( "Mutate" ) ) },
    { ... }
};